dist/app/modules/MainScreen/MainScreen.js

```diff
@@ -0,0 +1,18 @@
+import React, { PureComponent } from 'react';
+import { MainWrapper, MainTitle } from './styles';
+import { Button } from 'react-native';
+import Modules from '../index';
+class MainScreen extends PureComponent {
+    constructor() {
+        super(...arguments);
+        this.handleToSecondScreen = () => {
+            this.props.navigation.navigate(Modules.ProfileScreenModule.name);
+        };
+    }
+    render() {
+        return (React.createElement(MainWrapper, null,
```

Comment:
Why React.createElement?
Please ise jsx sytax only and remove all createElements from it.

Reply:
just remove dist :)